### PR TITLE
Update guidelines-that-support-professionals-use.md

### PR DESCRIPTION
**Pull request by theley502**
# Pull request guidance

Thank you for submitting your contribution to our support content! Our team works closely with subject matter experts in CSS and PMs in the product group to review all content requests to ensure technical accuracy and the best customer experience. This process can sometimes take one or more days, so we greatly appreciate your patience.

We also need your help in order to process your request as soon as possible:

- We won't act on your pull request (PR) until you type "**#sign-off**" in a new comment in your pull request (PR) to indicate that your changes are complete.

- After you sign off in your PR, the article will be tech reviewed by the PM or SME if it has more than minor changes. Once the article is approved, it will undergo a final editing pass before being merged.

### CODE REVIEW

**Comment by lucciz01** [comment]
Hi @theley502, I made a few changes based on your updates. Could you have a look?

File: support/dynamics/gp/guidelines-that-support-professionals-use.md [hunk 1]
```diff
@@ -16,17 +16,15 @@ _Original KB number:_ &nbsp; 850201
 
 Microsoft support professionals use to determine when a support case becomes a consulting engagement for the following items:
```
Comment (aartig13) [inline support/dynamics/gp/guidelines-that-support-professionals-use.md:17]: @lucciz01 please edit this sentence... it is not complete.

Comment (lucciz01) [reply support/dynamics/gp/guidelines-that-support-professionals-use.md]: Hi @aartig13, I moved the first sentence into the Introduction section. You can find my changes in the **Files changed** tab of this PR. Loop @theley502 for further suggestions.

Comment (aartig13) [reply support/dynamics/gp/guidelines-that-support-professionals-use.md]: Thanks! Looks good

**Comment by lucciz01** [comment]
Merged this PR per the confirmation from @theley502 in Teams Chat.

**Pull request merged**